Frameworks/CoreText/CTLine.mm

```diff
@@ -322,8 +347,7 @@ CGRect CTLineGetImageBounds(CTLineRef line, CGContextRef context) {
  @Status Interoperable
 */
 double CTLineGetTypographicBounds(CTLineRef lineRef, CGFloat* ascent, CGFloat* descent, CGFloat* leading) {
-    _CTLine* line = static_cast<_CTLine*>(lineRef);
-
+    __CTLine* line = const_cast<__CTLine*>(lineRef);
```

Comment:
Also we need the cast anyways, since this method is modifying lineRef. so we can't do lineRef->GetTypographics...(), given GetTypographics can't be a const method.

---
In reply to: [112746739](https://github.com/Microsoft/WinObjC/pull/2586#discussion_r112746739) [](ancestors = 112746739,112746377)